src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java

```diff
@@ -94,7 +108,52 @@
  *   &lt;property name="maximum" value="10"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * &#47;**
+ * * This file is copyrighted under CC. // Ok, as the file contains a copyright statement.
+ * *&#47;
+ * class MyClass {
+ *
+ * }
+ * </pre>
+ * <pre>
+ * &#47;** // violation, as the file doesn't contain a copyright statement.
+ * * MyClass as a configuration example.
+ * *&#47;
+ * class MyClass {
  *
+ * }
+ * </pre>
+ * <p>
+ *  An example of how to configure the check to make sure sql files contains the term 'license'.
+ * </p>
+ * <pre>
+ * &lt;module name="RegexpSingleline"&gt;
+ *     &lt;property name="format" value="license"/&gt;
+ *     &lt;property name="minimum" value="1"/&gt;
+ *     &lt;property name="maximum" value="10"/&gt;
```

Comment:
why 10 ?
can we avoid this parameter?

Reply:
I think if we specify a minimum we must specify a maximum as well since the default value for the maximum is 0.

Reply:
@romani will you review my comments?

Reply:
please make it 9999 to make it clear we do not care about max occurrences.

Reply:
done